sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java

```diff
@@ -124,6 +124,13 @@ public interface KubernetesCluster
      */
     String agentPoolResourceGroup();
 
+    /**
+     * Whether the kubernetes cluster can be accessed from public network.
+     *
+     * @return whether the kubernetes cluster can be accessed from public network.
+     */
+    boolean isPublicNetworkAccessEnabled();
```

Comment:
I think I made it clear in the sample PR, the getter is
```
PublicNetworkAccess publicNetworkAccess()
```
If the module has such class.